packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -864,6 +958,21 @@ describe.each([
             query: [
               { operator: "equal", field: "newField", value: "thatValue" },
             ],
+            // Should also update queryUI because query was not previously set.
+            queryUI: {
+              groups: [
+                {
+                  logicalOperator: "all",
+                  filters: [
+                    {
+                      operator: "equal",
```

Comment:
Sometimes the enum `BasicOperator` is used and sometimes its a string here, is that intended?

Reply:
No 🙈